src/main.ts

```diff
@@ -89,7 +89,7 @@ export class Main {
         storageDefaults[ConstantsService.vaultTimeoutActionKey] = 'lock';
         this.storageService = new ElectronStorageService(app.getPath('userData'), storageDefaults);
 
-        this.windowMain = new WindowMain(this.storageService, true);
+        this.windowMain = new WindowMain(this.storageService, true, 950, 600, (arg) => this.processDeepLink(arg));
```

Comment:
Do we need to pass these size demensions? What were they defaulting to in jslib?

Reply:
These are the default dimens for the constructor. Silly angular question: if there are defined, but I need to pass the action to fire the protocol which is the 4th param, can I just construct it like this:

` new WindowMain(this.storageService, true, (arg) => etc etc)` ?

Reply:
@vincentsalucci , you should be able to use `undefined` in place of those values, see: https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters, since it's beyond Angular and more around TypeScript when calling with those additional optional parameters.

e.g.: 
```typescript
this.windowMain = new WindowMain(this.storageService, true, undefined, undefined, (arg) => this.processDeepLink(arg));
```

Reply:
@cscharf - Thank you!!! Will update.

Reply:
For reference - this is the superclass constructor in `jslib`

<img width="599" alt="Screen Shot 2020-08-08 at 5 46 34 PM" src="https://user-images.githubusercontent.com/26154748/89721111-40e50600-d99f-11ea-864a-27d64ad5c09e.png">